Utilities/changing_names_between_locales.py

```diff
@@ -0,0 +1,115 @@
+                    for word, pos in tagged_sent:
+                        if pos == 'NNP':
+                            word = word.replace("'s", '')
+                            if word.isalpha():
+                                propernouns.add(word)
+        # List of proper nouns
+        print(propernouns)
+        # Creating a new file named fout_fname to store proper nouns
+        with open(fout_fname, 'w', encoding="utf8") as f:
+            for item in propernouns:
+                print(item, file=f)
+
+
+def merge_proper_nouns_files(fin_files, fout_file):
+    """
+    This function get all the  proper nouns and generate fout_file file which
+    will contains all the unique proper nouns
+    fin_files: List of the files which contains proper nouns
+    fout_file: Output file which will finally contains proper nouns(not opened
+    in append mode)[*.txt file]
+    Example:
+        merge_proper_nouns_files(["propernouns.txt", "propernouns2.txt"], 'merged_names2.txt')
+    """
+    propernouns = set()
+    # Reading each of the proper nouns from the each of the txt file
+    for file in fin_files:
+        f = open(file, "r", encoding="utf8").readlines()
+        for line in f:
+            word = line.strip()
+            propernouns.add(word)
+    # Creating a new file which stores all the unique proper nouns
+    print(propernouns)
+    with open(fout_file, 'w', encoding="utf8") as f:
+        for item in propernouns:
+            print(item, file=f)
+
+
+def replace_proper_nouns(tsv_in, tsv_out, map_fname):
+    """
+    This function will change the names in TSV and generate a new file which
+    will contains new names
+    tsv_in: The TSV file where the names are to be changed
+    tsv_out: The TSV file that will be generated which will contains the
+            changed name
+    map_fname: This file will contains tab separated two values as the mapping
+            from the old name to new name
+    Example:
+        replace_proper_nouns('wordwindow_level_en.tsv', 'wordwindow_level_en_change.tsv', 'mapping.txt')
+    """
+    # Reading each line from the mapping file and creating a map for the old name and new name
+    f_indian = io.open(map_fname, 'r', encoding='utf-8').readlines()
+    names_dict = dict()
+    for line in f_indian:
+        src_word, dst_word = line.split('\t')
+        src_word = src_word.strip()
+        dst_word = dst_word.strip()
+        names_dict[src_word] = dst_word
+
+    # Reading the rows from the SRC TSV file and updating the cells and writing the new cells into DEST TSV file
+    with open(tsv_in, 'r',  newline='', encoding='utf-8-sig') as fin, open(tsv_out, 'w', newline='', encoding='utf-8') as fout:
+        reader = csv.reader(fin, delimiter='\t', quoting=csv.QUOTE_NONE)
+        writer = csv.writer(fout, delimiter='\t', quotechar='', quoting=csv.QUOTE_NONE)
+        for line in reader:
+            length = len(line)
+            for i in range(length):
```

Comment:
You can write len(line) in range function as it is of O(1) complexity and won't do any harm